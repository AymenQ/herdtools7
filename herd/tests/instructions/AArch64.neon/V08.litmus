AArch64 V08
(* Tests load / store immediate, with offset *)
{ 
0:X0=x; 0:X2=y;
}

 P0                 ;
 MOV X1, #1         ;
 STR X1, [X0, #8]   ;
 LDR B0, [X0, #8]   ;
 STR B0, [X2]       ;
 forall (y=1)
