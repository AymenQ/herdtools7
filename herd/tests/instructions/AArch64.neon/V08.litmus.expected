Test V08 Required
States 1
y=1;
Ok
Witnesses
Positive: 1 Negative: 0
Condition forall (y=1)
Observation V08 Always 1 0
Hash=715852669a23ac0aaa1d902bda99fa08

